src/tests/ftest/ior/ior_small.py

```diff
@@ -36,7 +36,7 @@ def test_ior_small(self):
 
         :avocado: tags=all,pr,daily_regression
         :avocado: tags=hw,large
-        :avocado: tags=daosio,checksum,iorsmall,mpich
+        :avocado: tags=daosio,checksum,iorsmall,mpich,dfuse
```

Comment:
Another one that was not formatted correctly before. I would think 39 and 40 here should be:
```
:avocado: tags=daosio,checksum,mpich,dfuse,DAOS_5610
:avocado: tags=iorsmall
```